spec/services/patient_deduplication/strategies_spec.rb

```diff
@@ -79,5 +79,19 @@
       expect(described_class.identifier_excluding_full_name_match(limit: 1).count).to eq 1
       expect(described_class.identifier_excluding_full_name_match.count).to eq 2
     end
+
+    it "scopes results to facilities" do
+      patient = create(:patient, full_name: "Patient one")
+      patient_passport_id = patient.business_identifiers.first.identifier
+
+      patient_dup = create(:patient, full_name: "Patient one dup")
+      patient_dup.business_identifiers.first.update(identifier: patient_passport_id)
+
+      other_facility = create(:facility)
+
+      expect(described_class.identifier_excluding_full_name_match(facilities: [patient.registration_facility]).count).to eq 1
+      expect(described_class.identifier_excluding_full_name_match(facilities: [patient_dup.registration_facility]).count).to eq 1
+      expect(described_class.identifier_excluding_full_name_match(facilities: [other_facility]).count).to eq 0
+    end
```

Comment:
In addition to my other comments, can we add the relevant test coverage to the controller specs as well?